vernac/declaremods.ml

```diff
@@ -715,24 +715,25 @@ let start_module export id args res fs =
       let typs, ctx = build_subtypes env mp params resl in
       None, typs, ctx
   in
-  let () = Global.push_context_set ~strict:true ctx in
+  let () = Global.push_context_set ~strict:true ctx' in
+  mp, res_entry_o, subtyps, params, ctx, ctx'
```

Comment:
```suggestion
  mp, res_entry_o, subtyps, params, Univ.ContetxSet.union ctx ctx'
```

Reply:
I didn't do that because we will sometimes perform the union for no reason, but if you think it is acceptable, I'm fine with the change.